src/common/transformations/include/transformations/common_optimizations/mark_exp_reduceop_to_keep_in_mixed_precision.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Please combine all mark precision sensitive into the common header file (and may be implementation into the common cpp file). To have separate file is not convenient.